cytominer_database/ingest_incl_parquet.py

```diff
@@ -0,0 +1,203 @@
+
+| plate_a/
+|   set_1/
+|       file_1.csv
+|       file_2.csv
+|       ...
+|       file_n.csv
+|   set_2/
+|       file_1.csv
+|       file_2.csv
+|       ...
+|       file_n.csv
+|   ...
+|   set_m/
+|       file_1.csv
+|       file_2.csv
+|       ...
+|       file_n.csv
+
+Example::
+
+    import cytominer_database.ingest
+
+    cytominer_database.ingest.seed(source, target, config_path)
+
+    where
+    source = path/to/plate_a
+    target = path/to/output/directory
+    config_path = path/to/plate_a/ingest_config.ini
+Notes: 
+* seed() will run regardless of where the config-file was stored, as long as its' full path is passed. 
+This is not the case for the command-line command, where the config-file is specified only by the basename 
+(e.g. ingest_config.ini) and is assumed to reside in path/to/plate_a.
+
+* 'file_1.csv', ..., 'file_n.csv' is typically equivalent to 'Cells.csv', 'Cytoplasm.csv', 'Nuclei.csv', 'Image.csv', 'Object.csv'.
+
+"""
+
+"""
+* Extended explanations of how the PARQUET writer is used (see also the readme.rst).
+[Big Picure]
+When opening a Parquet file with the Pyarrow.parquet.writer, we use a reference table to set
+one fixed table schema (pyarrow.Table.schema) (per table kind).
+[Schema compatibility: column names]
+For each new table to be written to the open Parquet writer, schema compatibility is assured by aligning the dataframes. 
+The alignment assures that the new table will hold all columns present in the reference table, however, columns not present in the 
+reference table will be dropped from the new table. 
+Hence it is important to assure that the reference table has all columns that will be present in any of the tables of its kind. 
+
+[Schema compatibility: column types]
+Furthermore, the column types of columns with identical column names must agree. This has proven to be problematic
+when software further up the processing pipeline casts the types of its' output automatically. 
+The easiest solution for this is to convert all values in the .csv table to strings, however, this can be a disadvantage in the 
+processing tasks futher down the pipeline. The type incompatipility occurs infrequently though,
+ e.g. when a 0-valued column is cast as 'int', instead of 'float', which makes the table schema incompatible with
+  the table schema from files with non-zero values in the same column with a 'float'-type). 
+ Hence it is in our experience sufficient to convert integers to float-types. 
+ The type conversion is applied explicitly and directly to both the reference table before opening the writer,
+  and to any table before appending it to the open writer. 
+ The conversion type is set in the config file in the schema section, with key type_conversion.
+  Possible values are 'int2float' and 'all2string'. We do not use any automatic type-hierarchy-based casting
+ (although this code was developed in the preliminary colaboratory notebook). 
```

Comment:
```suggestion
```

We don't need to mention this in production software

Reply:
Was removed